backend-zs/controllers/stories.controller.js

```diff
@@ -11,4 +15,70 @@ exports.getStory = function (req, res) {
     Story.findById(req.params.id)
         .then(story => res.json(story))
         .catch(error => res.status(500).json({ error: error.message }))
-}
\ No newline at end of file
+}
+
+exports.deleteStory = function (req, res) {
+    Story.findByIdAndDelete({ _id: req.params.id })
+        .then(deletedStory => {
+            if (!!deletedStory.authorImage) {
+                S3.deleteElements([deletedStory.authorImage], process.env.S3_BUCKET_NAME)
+            }
+            res.json('story with id ' + deletedStory._id + ' was deleted successfully')
+        })
+        .catch(error => res.status(500).json({ error: error.message }))
+}
+
+exports.createStory = function (req, res) {
+    singleUpload(req, res, function (err) {
+        if (err) {
+            return res.status(500).json({ err: err.message })
+        }
+
+        let authorImage = !!req.file ? req.file.key : ""
+        let convertedStoryPages = convertStoryToPages(req.body.storyPages, MIN_CHARACTERS_PAGE)
+        let story = new Story({
+            title: req.body.title,
```

Comment:
we might send the story object in the with json key `story` `{'story':{'title': ...}}` and create the story directly from it

Reply:
Will this work if we are sending the body as a form-data?